cadence/contracts/AxelarGateway.cdc

```diff
@@ -0,0 +1,312 @@
+import AxelarAuthWeighted from "./auth/AxelarAuthWeighted.cdc"
+import Crypto
+
+// Main GateWay contract for audit
+access(all) contract AxelarGateway {
+  access(self) let SELECTOR_APPROVE_CONTRACT_CALL: [UInt8]
+  access(self) let SELECTOR_TRANSFER_OPERATORSHIP: [UInt8]
+  access(all) let PREFIX_APP_CAPABILITY_NAME: String
+
+  access(all) resource CGPCommand {
+    access(all) var commandId: String
+    access(all) var isExecuted: Bool
+
+    destroy() {
+      emit Executed(commandId: self.commandId)
```

Comment:
I'd recommend emitting this in an `access(contract)` method dedicated to destroying this resource. Only because there's currently an ongoing language design discussion about removing custom destructors and this may need to be removed in the near future anyway.